pkg/inspector/image-inspector_test.go

```diff
@@ -17,6 +17,9 @@ type SuccMockScanner struct {
 type NoResMockScanner struct {
 	SuccMockScanner
 }
+type SuccWithHTMLMockScanner struct {
```

Comment:
@enoodle is this OK with `gofmt`? I have the feeling that you should leave an empty line between the two definitions.